src/api/views.py

```diff
@@ -32,7 +32,9 @@
         "Read more about choosing your .gov domain.</a>".format(public_site_url("domains/choosing"))
     ),
     "invalid": "Enter a domain using only letters, numbers, or hyphens (though we don't recommend using hyphens).",
-    "success": "That domain is available!",
+    "success": "That domain is available! We’ll try to give you the domain you want, \
```

Comment:
**Question (nonblocking):** Would we also want to change the "success" wording on the 11ty website as well for the availability button?

Reply:
> **Question (nonblocking):** Would we also want to change the "success" wording on the 11ty website as well for the availability button?

Good callout! I don't think we'd make the message exactly the same because the user won't be in the midst of completing the form, but we can clarify the difference between "available" and "you're getting it!"